tests/port_dpb.py

```diff
@@ -225,7 +225,11 @@ def verify_asic_db(self):
             assert(fvs_dict['SAI_PORT_ATTR_HW_LANE_LIST'] == self.get_lanes_asic_db_str())
         assert(fvs_dict['SAI_PORT_ATTR_SPEED'] == str(self.get_speed()))
 
+    def verify_breakout_mode(self, breakout_mode):
+        self._dvs.get_config_db().wait_for_field_match("BREAKOUT_CFG", self.get_name(), {"brkout_mode": breakout_mode})
```

Comment:
Ah okay yeah, this is what I suggested above. 😆 Why do we need it in both `DockerVirtualSwitch` and `DPB`?

Reply:
Initially when I wrote it, I wrote it in DockerVirtulSwitch, later I wrote one in DPB, as I feel thats the right place.

But I still hesitate to call the one in DPB, because it instantiates a Port object using port_name, where a lot of things happen, and then it calls verify_breakout_mode in Port class. So, doing all that to just check port breakout mode seems a bit too much.

But if you really think, debatably, port_breakout_mode should be a field of port entry in PORT_TABLE in config DB. In which case all the above makes sense.

Anyways, for now, I will change the one in DPB and use it everywhere, delete the one in DockerVirtualSwitch.